# Patch auto brightness call
#    
# Fixes crash due to nonexient system library
# `setOplusAutoBrightnessAdjustment() - Set allies1 auto brightness failed`
#
# The app checks whether the device is a oneplus device by reading build.prop.
# If that is the case, it tries to enable auto-brightness feature and crashes, because
# the needed system library is not installed.
#
# This patch reverts the first check, instead of activating the fetaure
# on a supported device, it returns immediately.
#
--- a/onepluscamera/smali/com/oneplus/camera/CameraActivity.smali
+++ b/onepluscamera/smali/com/oneplus/camera/CameraActivity.smali
@@ -6564,7 +6564,7 @@
 
     const-string v5, "OwnerId"
 
-    if-eqz v3, :cond_0
+    if-nez v3, :cond_0
 
     .line 1936
     move-object v3, p0
